ihmc-java-toolkit/src/main/java/us/ihmc/tools/thread/MissingThreadTools.java

```diff
@@ -10,6 +11,24 @@
 
 public class MissingThreadTools
 {
+   /**
+    * {@link ThreadTools#sleepSeconds} can actually return early.
```

Comment:
How did you realize this? Looking at the code for `ThreadTools#sleep()` it seems to do essentially the same thing as this method (sleep, see if it slept long enough, if not sleep again). I'd be surprised if it somehow  returned early.

Reply:
Yup it was really the other sleepSeconds method cutting off the subnanosecond part, allowing it to undersleep by a nanosecond at most.